WordPress/src/main/java/org/wordpress/android/util/config/SelfHostedUsersFeatureConfig.kt

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.Feature
+import javax.inject.Inject
+
+/**
+ * Configuration of the self-hosted users feature
+ */
+private const val ENABLE_SELF_HOSTED_USERS_REMOTE_FIELD = "enable_self_hosted_users"
```

Comment:
Admittedly a nitpick, but the term "enable" feels redundant within a flag name and it appears all other existing flags do not include a verb, only subjects — i.e., `some_feature` rather than `enable_some_feature` or `allow_some_feature`. 

Please feel free to ignore this suggestion as it has little-to-no impact. 😄 

```suggestion
private const val ENABLE_SELF_HOSTED_USERS_REMOTE_FIELD = "self_hosted_users"
```

Reply:
That's a fair point. I'll make the change!